cmd/app/link_test.go

```diff
@@ -430,7 +431,7 @@ func Test_Apps_Link(t *testing.T) {
 			CmdArgs:       []string{},
 			ExpectedError: slackerror.New(slackerror.ErrAppNotFound),
 		},
-		"accept manifest source prompt and saves information about the provided deployed app": {
+		"accepting manifest source prompt should save information about the provided deployed app": {
```

Comment:
note: Just re-wording after reading over the test cases for clarity.

Reply:
```
: {
```

Reply:
```
: }
```